### PR TITLE
Remove vulnerabilities and improve some doc

### PR DESCRIPTION
**Pull request by joshdholtz**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -847,7 +847,7 @@ <h2 class="section__headline">Become a fastlane contributor</h2>
       <div class="footer__logo">
         <a target="_blank" href="https://developers.google.com/"><img class="footer__logo__img" src="/assets/images/google-developers-lockup-color.svg" alt="Google Developers logo"></a>
       </div>
-      <span class="footer__copyright">© 2018 Google, Inc. All Rights Reserved</span>
+      <span class="footer__copyright">© 2017 Google, Inc. All Rights Reserved</span>
```
Comment (snatchev) [inline index.html]: 2017?

File: index.html [hunk 2]
```diff
@@ -380,12 +380,12 @@ <h2 class="section__headline">Capture screenshots automatically</h2>
...
       <div class="feature__flex">
         <div class="feature__content-wrapper">
-          <h5>Generate localized app store screenshots in minutes.</h5>
+          <h5 id="generate-localized-app-store-screenshots-in-minutes.">Generate localized app store screenshots in minutes.</h5>
 <ul>
 <li>Automatically capture localized screenshots for each language and device
 your app supports</li>
 <li>Configure it once, and store the configuration so anyone on the team
-can generate screenshots</li>
+generate screenshots</li>
```
Comment (snatchev) [inline index.html]: I think we need the verb here.

File: index.html (hunk 1 above)
Comment (joshdholtz) [reply index.html]: OH WHAT?!

File: index.html (hunk 2 above)
Comment (joshdholtz) [reply index.html]: Totes need verb

File: index.html (hunk 1 above)
Comment (joshdholtz) [reply index.html]: How did that even happen? 🤔

**Pull request merged**